extensions/mssql/package.json

```diff
@@ -494,7 +494,7 @@
         },
         {
           "command": "mssql.newObject",
-          "when": "connectionProvider == MSSQL && nodeType == Folder && objectType =~ /^(ServerLevelLogins|Users|ServerLevelServerRoles|ApplicationRoles|DatabaseRoles)$/ && config.workbench.enablePreviewFeatures",
+          "when": "connectionProvider == MSSQL && nodeType == Folder && objectType =~ /^(ServerLevelLogins|Users|ServerLevelServerRoles|ApplicationRoles|DatabaseRoles|Databases)$/ && config.workbench.enablePreviewFeatures",
```

Comment:
Please remove these until the feature is actually complete - the rest of the PR can stay (as long as it doesn't cause any problems with existing functionality) but we shouldn't have incomplete/broken experiences showing to users using insiders

Reply:
I'll remove them before merging the PR if I haven't gotten the tools service changes to work by EOD.

Reply:
Given that you still have to get the STS changes reviewed, then get an STS build and update it here I'd say just assume you'll be getting these changes in first and remove them now. This lets us focus on the changes currently in this PR and getting it in earlier helps ensure we aren't breaking anything else that Alan & co are working on.

You can do a simple follow up PR with adding these back in + STS update once that's all ready - no problems there.

Reply:
Done.